### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by andrew17752985**
https://andrew17752985.github.io/react_goods-selector-js/

### CODE REVIEW

**Review by mykhalenych — CHANGES_REQUESTED** [review]
https://github.com/mate-academy/react_goods-selector-js/assets/53228259/b0c9829e-2ed2-40e8-8179-cda0be01eee4

https://mate-academy.github.io/react_goods-selector/
check example and fix your code

File: src/App.jsx [hunk 1]
```diff
@@ -14,70 +15,71 @@ export const goods = [
...
-              type="button"
-              className="button"
+      <table className="table">
+        <tbody>
+          {goods.map(good => (
+            <tr
+              key={good}
+              data-cy="Good"
+              className={
+                selectedGood === good ? 'has-background-success-light' : ''
```
Comment (volodymyr-soltys97) [inline src/App.jsx:50]: Create a variable for this check `selectedGood === good` and use them everywhere

File: src/App.jsx [hunk 2]
```diff
@@ -14,70 +15,71 @@ export const goods = [
...
-              +
-            </button>
-          </td>
+              <td>
+                {selectedGood === good ? (
+                  <button
+                    data-cy="RemoveButton"
+                    type="button"
+                    className="button is-info"
+                    onClick={() => clearSelection()}
```
Comment (volodymyr-soltys97) [inline src/App.jsx:59]: There is no need to create another function in onClick, you can immediately use the function you created earlier
```suggestion
                    onClick={clearSelection}
```

**Review by volodymyr-soltys97 — APPROVED** [review]
Well done 🔥

**Pull request closed without merging**